Assets/Countly/Plugins/CountlySDK/Services/UserDetailsCountlyService.cs

```diff
@@ -24,54 +23,188 @@ internal UserDetailsCountlyService(CountlyConfiguration configuration, CountlyLo
+        /// <param name="key">string with property name to increment</param>
+        /// <param name="value">double value by which to increment</param>
+        public void IncrementBy(string key, double value)
+        {
+            Log.Info("[UserDetailsCountlyService] IncrementBy," + " key: " + key + " value: " + value);
+            IncrementInternal(key, value);
+        }
+
+        /// <summary>
+        /// Multiply custom property value by provided value.
+        /// </summary>
+        /// <param name="key">string with property name to multiply</param>
+        /// <param name="value">double value by which to multiply</param>
+        public void Multiply(string key, double value)
+        {
+            Log.Info("[UserDetailsCountlyService] Multiply," + " key: " + key + " value: " + value);
+            MultiplyInternal(key, value);
+        }
+
+        /// <summary>
+        /// Save maximal value between existing and provided.
+        /// </summary>
+        /// <param name="key">String with property name to check for max</param>
+        /// <param name="value">double value to check for max</param>
+        public void Max(string key, double value)
+        {
+            Log.Info("[UserDetailsCountlyService] Max," + " key: " + key + " value: " + value);
+            MaxInternal(key, value);
+        }
+
+        /// <summary>
+        /// Save minimal value between existing and provided.
+        /// </summary>
+        /// <param name="key">string with property name to check for min</param>
+        /// <param name="value">double value to check for min</param>
+        public void Min(string key, double value)
+        {
+            Log.Info("[UserDetailsCountlyService] Min," + " key: " + key + " value: " + value);
+            MinInternal(key, value);
+        }
+
+        /// <summary>
+        /// Create array property, if property does not exist and add value to array
+        /// You can only use it on array properties or properties that do not exist yet.
+        /// </summary>
+        /// <param name="key">string with property name for array property</param>
+        /// <param name="value">array with values to add</param>
+        public void Push(string key, string[] value)
+        {
+            Log.Info("[UserDetailsCountlyService] Push," + " key: " + key + " value: " + value);
+            PushInternal(key, value);
+        }
+
+        /// <summary>
+        /// Create array property, if property does not exist and add value to array, only if value is not yet in the array
+        /// You can only use it on array properties or properties that do not exist yet.
+        /// </summary>
+        /// <param name="key">string with property name for array property</param>
+        /// <param name="value">array with values to add</param>
+        public void PushUnique(string key, string[] value)
+        {
+            Log.Info("[UserDetailsCountlyService] PushUnique," + " key: " + key + " value: " + value);
```

Comment:
whenever you print a value please cover them with brackets:
"key:[" + key + "]"

this way empty values are easier to observe